pom.xml

```
@@ -144,7 +144,7 @@
             <dependency>
                 <groupId>com.okta.oidc.tck</groupId>
                 <artifactId>okta-oidc-tck</artifactId>
-                <version>0.5.9</version>
+                <version>0.5.10-SNAPSHOT</version>
```

TODO: Update after https://github.com/okta/okta-oidc-tck/pull/179 is merged.

```suggestion
                <version>0.5.10</version>
```